openssl.nuspec

```diff
@@ -1,37 +1,24 @@
      uppercase omega letter enclosed in quotation marks,
-	 you should use an editor that supports UTF-8, not this one. -->
+     you should use an editor that supports UTF-8, not this one. -->
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>openssl</id>
     <title>OpenSSL (for software developers)</title>
-    <version>1.0.2.20150731</version>
-    <authors>Alex Willmer</authors>
-    <owners>Alex Willmer</owners>
+    <version>1.0.2.20160504</version>
+    <authors>Shining Light Productions</authors>
+    <owners>Alex Willmer, Jesse Sanford</owners>
     <summary>The Open Source SSL and TLS toolkit, with development headers</summary>
-    <description>The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured, and Open Source toolkit implementing the Secure Sockets Layer (SSL v2/v3) and Transport Layer Security (TLS v1) protocols as well as a full-strength general purpose cryptography library. The project is managed by a worldwide community of volunteers that use the Internet to communicate, plan, and develop the OpenSSL toolkit and its related documentation.
-	This package include the development headers needed to compile software against OpenSSL.
+    <description>The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured, and Open Source toolkit implementing the Secure Sockets Layer (SSL v2/v3) and Transport Layer Security (TLS v1) protocols as well as a full-strength general purpose cryptography library. The project is managed by a worldwide community of volunteers that use the Internet to communicate, plan, and develop the OpenSSL toolkit and its related documentation. This package include the development headers needed to compile software against OpenSSL.
     </description>
     <projectUrl>https://slproweb.com/products/Win32OpenSSL.html</projectUrl>
     <packageSourceUrl>https://github.com/moreati/openssl-chocolatey</packageSourceUrl>
     <projectSourceUrl>https://www.openssl.org/source/</projectSourceUrl>
-    <!--<docsUrl></docsUrl>
-    <mailingListUrl></mailingListUrl>
-    <bugTrackerUrl></bugTrackerUrl>-->
     <tags>openssl SSL TLS pfx pem key RSA admin</tags>
-    <copyright></copyright>
     <licenseUrl>https://www.openssl.org/source/license.html</licenseUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <iconUrl>http://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/master/icons/openssl.png</iconUrl>
-    <!--<dependencies>
-      <dependency id="" version="__VERSION__" />
-      <dependency id="" />
-    </dependencies>-->
-    <releaseNotes></releaseNotes>
-    <!--<provides></provides>-->
+    <releaseNotes> 6/29/2016: Updated to mimic install of Openssl.Light package: https://chocolatey.org/packages/openssl.light</releaseNotes>
   </metadata>
-  <files>
-    <file src="tools\**" target="tools" />
-  </files>
```

Comment:
Ah, I think I understand now. I was overriding the extract dir by supplying `unzipLocation` to `Install-ChocolateyPackage`. This commit removes that override, hence there's no need to override it here.